config/config.go

```diff
@@ -13,19 +13,23 @@ import (
 
 var parsedConfig *SourcesApiConfig
 
+// KafkaSaslSettings is a struct containing the settings for authenticating to the managed Kafka instances.
+type KafkaSaslSettings struct {
+	CaPath           string
+	Mechanism        string
+	Password         string
+	SecurityProtocol string
+	Username         string
+}
```

Comment:
Digging through the code some more I'm starting to see why you did this. It's annoying that the ca path is separate from the rest of these...app-common-go's sasl config just contains the bottom four fields :rage: 

I wish we could just use their config :cry: 

---

I almost wonder if we should just use the [`clowder.BrokerConfig`](https://github.com/RedHatInsights/app-common-go/blob/6ca81c147285330f80af3384fa41397abfa9cdaa/pkg/api/v1/types.go#L404-L419) since that has all of the pieces we need. Then we wouldn't have to have our own struct, could just pass in the broker config from our config. 

Curious your thoughts here - it would save us having to parse it out etc.